### ISSUE TITLE
[Feature Request] Include Proton GE by default

### ISSUE THREAD

**Issue by btegs** [tkashkin/GameHub#346]
Right now, GameHub relies on having Steam already installed on your system and will reference that for using Proton. But what about those who don't have Steam installed and/or want more control over Proton?

Why not include [Proton GE](https://github.com/GloriousEggroll/proton-ge-custom) by default with installations of GameHub? We can still reference the system Proton if it exists, but it would be nice to have a community version of Proton always be there as a backup or alternative.

What do you think?

**Comment by JKingweb** [comment]
As someone who does not otherwise use Steam, I would really appreciate this.

**Comment by btegs** [comment]
@tkashkin - Have you considered this at all? What if some people do not have Steam installed and want to use Proton? Can always go the Proton GE option or grabbing the source for Proton itself and [build it](https://github.com/ValveSoftware/Proton/wiki/Build-instructions-(Ubuntu)).

**Comment by tkashkin** [comment]
Yes, I plan to support custom compatibility tools, including custom Proton versions.

I'm currently refactoring a large part of code so it'll take some time.

**Comment by btegs** [comment]
There's been a lot of work in the Proton GE front: https://github.com/GloriousEggroll/proton-ge-custom/releases so I would like to see if you have an update on this.

Maybe you should include a default directory for people to put their custom Proton in while still looking in the compatibilitytools.d in their Steam install if Steam exists. As @JKingweb mentioned, it would be nice to have Proton independent of Steam where we have a choice to drop in the official or community versions of Proton (Proton GE) in our GameHub directory.

**Comment by ShadwDrgn** [comment]
would love to see an update on this. GloriousEggroll is amazing. Even if it means having to grab from the steam compatibilitytools.d folder or w/e. Proton for use in non-steam games is massively interesting to me.
